certified-connectors/AzureAD/apiDefinition.swagger.json

```diff
@@ -438,6 +531,62 @@
     }
   },
   "definitions": {
+    "Get_Manager_Respons": {
+      "description": "Get manager respons object",
+      "type": "object",
+      "properties": {
+        "@@odata.context": {
+          "type": "string"
+        },
+        "@@odata.type": {
+          "type": "string"
+        },
+        "id": {
+          "type": "string",
+          "description": "Manager id."
+        },
+        "businessPhones": {
+          "type": "array",
+          "items": {
+            "type": "string",
+            "description": "Manager businessPhones"
```

Comment:
Perhaps, this "description" is related to "businessPhones" property instead of its items.

Reply:
done

Reply:
done